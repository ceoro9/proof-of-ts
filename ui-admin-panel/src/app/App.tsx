import React from 'react';

const App = () => {
	return (
		<div>Hello World!!!!!..h....o</div>
	);
};

export default App;
