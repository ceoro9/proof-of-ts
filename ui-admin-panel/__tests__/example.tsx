import React       from 'react';
import { shallow } from 'enzyme';
import App         from '../src/app/App';

describe('AppComponent', () => {
  it('should render', () => {
    const component = shallow(<App/>);
    expect(component).toMatchSnapshot();
  });
});
