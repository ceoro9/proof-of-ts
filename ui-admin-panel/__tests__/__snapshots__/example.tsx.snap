// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppComponent should render 1`] = `
<Fragment>
  <div
    className="app"
  >
    Hello World!!!!
  </div>
  <div
    className="nice"
  >
    Nice
  </div>
  <div
    className="golo"
  >
    Golo
  </div>
</Fragment>
`;
